### DISCUSSION TITLE
when will tracing 0.2 be released?

### DISCUSSION THREAD

**Discussion by bigdogs** [tokio-rs/tracing#1852] [Q&A] [answered]
Is there issue blocking the releasing?

**Comment by davidbarsky** [comment, marked as answer]
Lack of time, but it should be out sooner than later: https://github.com/tokio-rs/tracing/issues/1520#issuecomment-1001042095.

**Reply by theoparis** [reply]
Any updates on this? It's 2026 and it still hasn't been released. I still have to rely on the main branch at the moment 😕
